File: MPI/mpicheck/tests/functions/4.0/MPI_Get_count_c.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const MPI_Status *var_0;
    MPI_Datatype var_1;
    MPI_Count *var_2;
    int ret;
    /* calls */
    ret = MPI_Get_count_c(var_0, var_1, var_2);
    ret = PMPI_Get_count_c(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/4.0/MPI_Get_count_c.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0(10)
       INTEGER var_1
       INTEGER(KIND=MPI_COUNT_KIND) var_2
       INTEGER var_3
        call mpi_get_count_c(var_0, var_1, var_2, var_3)
       call pmpi_get_count_c(var_0, var_1, var_2, var_3)
        end program main
    